test/js/samples/action-custom-event-handler/expected.js

```diff
@@ -43,7 +43,7 @@ function handleFoo(bar) {
 
 function foo(node, callback) {
 	
-}
+} // code goes here
```

Comment:
It's unfortunate that these (and other similar comments) don't end up in the right place. This is a limitation of how code-red handles comments (I'm not sure there'd be anything to attach them to so they'd appear correctly inside the braces), but it seems kind of low priority.